cmd/crictl/main_unix.go

```diff
@@ -22,3 +22,5 @@ const (
 	defaultConfigPath      = "/etc/crictl.yaml"
 	defaultRuntimeEndpoint = "unix:///var/run/dockershim.sock"
 )
+
+var defaultRuntimeEndpoints = []string{"unix:///var/run/dockershim.sock", "unix:///run/containerd/containerd.sock", "unix:///run/crio/crio.sock"}
```

Comment:
looks like the string is set by default in the cli? https://github.com/kubernetes-sigs/cri-tools/pull/599/files#diff-20e8c63cd3f9c0ee9151d8e93c4ff9faR186

Reply:
It needs it because it sets `defaultRuntimeEndpoint` if EP is not set by CLI or config file: https://github.com/kubernetes-sigs/cri-tools/blob/master/cmd/crictl/main.go#L190.  Default CLI value: https://github.com/kubernetes-sigs/cri-tools/blob/master/cmd/crictl/main.go#L143

Reply:
`defaultRuntimeEndpoints[0]==defaultRuntimeEndpoint`

Reply:
:) if you are going to check `if defaultRuntimeEndpoints[0] || defaultRuntimeEndpoint` then you don't need to set the cli value to a default.. you don't need to do that because your default is the array.